bot/exts/evergreen/trivia_quiz.py

```diff
@@ -1,56 +1,236 @@
 import asyncio
 import json
 import logging
+import operator
 import random
 from pathlib import Path
+from typing import List, Optional, Tuple
 
 import discord
 from discord.ext import commands
 from fuzzywuzzy import fuzz
 
+from bot.constants import Colours
+from bot.constants import NEGATIVE_REPLIES
 from bot.constants import Roles
 
-
 logger = logging.getLogger(__name__)
 
+VARIATION_TOLERANCE = 83
 
 WRONG_ANS_RESPONSE = [
     "No one answered correctly!",
-    "Better luck next time"
+    "Better luck next time...",
+]
+
+N_PREFIX_STARTS_AT = 5
+N_PREFIXES = [
+    "penta", "hexa", "hepta", "octa", "nona",
+    "deca", "hendeca", "dodeca", "trideca", "tetradeca",
+]
+
+PLANETS = [
+    ("1st", "Mercury"),
+    ("2nd", "Venus"),
+    ("3rd", "Earth"),
+    ("4th", "Mars"),
+    ("5th", "Jupiter"),
+    ("6th", "Saturn"),
+    ("7th", "Uranus"),
+    ("8th", "Neptune"),
+]
+
+TAXONOMIC_HIERARCHY = [
+    "species", "genus", "family", "order",
+    "class", "phylum", "kingdom", "domain",
 ]
 
+UNITS_TO_BASE_UNITS = {
+    "hertz": ("(unit of frequency)", "s^-1"),
+    "newton": ("(unit of force)", "m*kg*s^-2"),
+    "pascal": ("(unit of pressure & stress)", "m^-1*kg*s^-2"),
+    "joule": ("(unit of energy & quantity of heat)", "m^2*kg*s^-2"),
+    "watt": ("(unit of power)", "m^2*kg*s^-3"),
+    "coulomb": ("(unit of electric charge & quantity of electricity)", "s*A"),
+    "volt": ("(unit of voltage & electromotive force)", "m^2*kg*s^-3*A^-1"),
+    "farad": ("(unit of capacitance)", "m^-2*kg^-1*s^4*A^2"),
+    "ohm": ("(unit of electric resistance)", "m^2*kg*s^-3*A^-2"),
+    "weber": ("(unit of magnetic flux)", "m^2*kg*s^-2*A^-1"),
+    "tesla": ("(unit of magnetic flux density)", "kg*s^-2*A^-1"),
+}
+
+
+def linear_system(q_format: str, a_format: str) -> Tuple[str, str]:
+    """Generate a system of linear equations with two unknowns."""
+    x, y = random.randint(2, 5), random.randint(2, 5)
+    answer = a_format.format(x, y)
+
+    coeffs = random.sample(range(1, 6), 4)
+
+    question = q_format.format(
+        coeffs[0],
+        coeffs[1],
+        (
+            coeffs[0] * x + coeffs[1] * y
+        ),
+        coeffs[2],
+        coeffs[3],
+        (
+            coeffs[2] * x + coeffs[3] * y
+        ),
+    )
+
+    return question, answer
+
+
+def mod_arith(q_format: str, a_format: str) -> Tuple[str, str]:
+    """Generate a basic modular arithmetic question."""
+    quotient, m, b = random.randint(30, 40), random.randint(10, 20), random.randint(200, 350)
+    ans = random.randint(0, 9)  # max 9 because min mod 10
+    a = quotient * m + ans - b
+
+    question = q_format.format(a, b, m)
+    answer = a_format.format(ans)
+
+    return question, answer
+
+
+def ngonal_prism(q_format: str, a_format: str) -> Tuple[str, str]:
+    """Generate a question regarding vertices on n-gonal prisms."""
+    n = random.randint(0, len(N_PREFIXES) - 1)
```

Comment:
You can use `random.randrange`

Reply:
alright, i'll change that

Reply:
wait actually, for consistency i won't. random.randint has been used throughout the entire code